bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tcomplex.py

```diff
@@ -0,0 +1,46 @@
+# Author: Massimiliano Galli CERN  05/2019
+# Original PyROOT code by Wim Lavrijsen, LBL
+
+################################################################################
+# Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from ROOT import pythonization
+import cppyy
+
+def _rsub(self, other):
```

Comment:
Even if it's super simple, can you add some docs to the helper methods that implement the pythonisations? Example:
https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_th1.py#L17